slamd/design_assistant/processing/design_assistant_service.py

```diff
@@ -216,4 +276,27 @@ def generate_design_knowledge(cls, token):
     @classmethod
     def generate_formulation(cls, design_knowledge, token):
         formulations = LLMService.generate_formulation(design_knowledge, token) 
-        return formulations
\ No newline at end of file
+        return formulations
+    
+    @classmethod
+    def get_template_of_selected_task(cls):
+        design_assistant_session = DesignAssistantPersistence.get_session_for_property('design_assistant')
+        if 'data_creation' in list(design_assistant_session.keys()): 
+            template = '/data_creation/powder_name.html'
+        if 'zero_shot_learner' in list(design_assistant_session.keys()):
+            template = '/zero_shot_learner/campaign_design_targets.html'
+        return template 
+    
+    @classmethod
+    def get_template_of_next_step(cls, material_type, progress):
```

Comment:
material_type wird hier nicht verwendet. Den Param brauchst du also gar nicht in der Methode